### DISCUSSION TITLE
Issues with SO install + accessing the web manager

### DISCUSSION THREAD

**Discussion by zmagez** [Security-Onion-Solutions/securityonion#10136] [Unsupported Versions]
This is my second time trying to install SO on my VM network and I am having multiple issues as this is the first time I am using it. I got it installed the first time, and couldn't figure out how to access the web manager or set up SO with my VM network. (This is for a class btw)
So I decided to do a fresh install.
(My SO version is 2.3.200)
During the fresh Install, I have a NAT nic and a bridged nic, but when I go to select the management nic, it states that the NAT nic is "link down" and failed preflight checks, and will be stuck on the install. 

I also don't know how to correctly configure SO for my VM network. My professor told me to switch the bridge nic to my VM net after the install, but I haven't figured out how to link it with my firewall box. Here's my network currently:

Host network:
192.168.1.0/24
Gateway: 192.168.1.1
(network adapter for vm net) 192.168.2.254

VM net:
192.168.2.0/24
Gateway: 192.168.2.1 (pfsense)
192.168.2.2-5 (windows + linux machines)

I would appreciate all the help I can get, I've watched videos for the install and documents on getting it to run with pfsense, but I just need the minor details on what exactly to do to fix this "link down", as well as how to actually link it with my vm network corrently.
Thank you.

**Comment by InfosecGoon** [comment]
I'm sorry, I'm having a little bit of trouble following your network topology.

You have two network adapters in the VM that you've installed Security Onion on -- one is a NAT interface, one is a Bridged interface. Which of these networks is the NAT interface on? Is the management interface for the Security Onion box behind the PFSense firewall (2.0/24 network) or the Host network (1.0/24)? And what are you using as your hypervisor platform?

**Reply by zmagez** [reply]
I am on VMware, the NAT is through my host network (1.0/24). I will provide some images to help give a better understanding. 
![b1a8cb036a2a513132051ef9150c86f8](https://user-images.githubusercontent.com/51929835/232597298-cd02159d-bf6d-4525-9f16-cc628a6dc0f1.png) 
(SO Adapters)
![5187fbfdec434382d7767958deb7528f](https://user-images.githubusercontent.com/51929835/232597435-7b99a336-8089-490a-8263-4f94d73b4cf1.png)
(NAT adapter)
![aacaeaf25b13ef713503dce8617254a4](https://user-images.githubusercontent.com/51929835/232597528-08c3e56f-a4a1-4f2f-a6b8-4fa6a4a9150f.png)
(VM networks)


I just tried to do another install, it finally said that my NAT nic was UP, but it still failed preflight checks, and gets stuck on 2% on the last install page. 
![09e3cbe7ba780477b54796fabc19a363](https://user-images.githubusercontent.com/51929835/232597743-4b6dac15-b0d0-4a68-b4e8-12b333c71d9e.png)

And for the management interface, I don't exactly know how to set that up, but I was told that for the install you have to have a bridge nic and after the install to change it to the VM net and then link it to the pfsense box.

**Reply by InfosecGoon** [reply]
I think we might have a bit of a vocabulary problem here -- let me define some of the terms so we're on the same page.

Management Interface -- this is the interface on your Security Onion standalone installation that has an IP address assigned to it and is used for active interaction with other endpoints on your network. For example, the management interface is where an analyst would connect to use the web console, or where an endpoint would forward logs for ingestion.

Monitor Interface(s) -- this/these are the interfaces that are configured in promiscuous mode to listen to traffic on the network. It does not have an IP address attached to it.

For your VMWare installation, you'll want to have a network interface that can have an IP address attached to it and connect to the Internet (for your Management Interface) and you'll want one that's on the bridge network that you want to monitor (that's your Monitor Interface).

So given what you've outline above, you'll want to put one on VmNet3 and one on the VM network, whatever you've named it. It's important that the one that you designate as the Management Interface has Internet access, or else you'll fail the preflight check, which seems to be what's happening here.
